tips.json

```diff
@@ -1,52 +1,87 @@
 [
-  { 
+  {
     "tool": "GitHub",
     "toolSlug": "github",
     "source": "https://help.github.com/articles/using-keyboard-shortcuts/",
     "text": "When you're browsing a repo on GitHub, pressing the [t] key will open up a search box and allow you to search for any file within the repo that you can then navigate to. This saves a ton of clicks especially when trying to find a file that's in a deeply-nested directory! Cool, right?"
   },
-  { 
+  {
     "tool": "GitHub",
     "toolSlug": "github",
     "source": "https://help.github.com/articles/using-keyboard-shortcuts/",
     "text": "Typing [g i] will bring you directly to all of your GitHub issues. Same with [g p] for pull requests! Gnarly!"
   },
-  { 
+  {
+    "tool": "Git",
+    "toolSlug": "git",
+    "source": "https://git-scm.com/book/en/v2/Git-Basics-Git-Aliases",
+    "text": "Create your own aliases in your [.gitconfig] file, so you don't need to type long command lines. Show others how to deploy with a simple [>git deploy]!"
+  },
+  {
     "tool": "Command Line",
     "toolSlug": "terminal",
     "text": "If you've made a typo in a command you haven't yet run, you can use [ctrl + a] to jump the cursor to the beginning of the command. [ctrl + e] will jump the cursor to the end. [option + click] will bring the cursor to wherever you clicked!"
   },
-  { 
+  {
     "tool": "Command Line",
     "toolSlug": "terminal",
     "text": "Use the [UP arrow] to cycle through previously run commands. This is useful when you're doing repetitive tasks like pushing commits to GitHub!"
   },
-  { 
+  {
     "tool": "Command Line",
     "toolSlug": "terminal",
     "text": "Use [ctrl + l] to clear the screen. This doesn't lose any of your previous commands, but can help if you need a visual break! Neat, huh?"
   },
   {
+    "tool": "Command Line",
+    "toolSlug": "terminal",
+    "text": "Use [ctrl + a] to go to the beginning of the line without losing any characters present on the command line."
+  },
+  {
+    "tool": "Command Line",
+    "toolSlug": "terminal",
+    "text": "Use '!!' to repeat your previous command. Because we love repeating ourselves right?"
```

Comment:
Can you wrap the !! in hard brackets instead of single quotes?

e.g. `[!!]`

This is the convention I've put in place (for now) for denoting commands, files and keys to be pressed. They'll get parsed and styled differently than the rest of the text based on being wrapped in those hard brackets.